src/upload/upload.controller.ts

```
@@ -88,11 +86,26 @@ export class UploadController {
+  @Get(':id')
+  @ApiOperation({
+    summary: '문서 단건 조회 (상태 포함)',
+    description: '업로드한 문서의 처리 상태를 조회합니다.',
+  })
+  @ApiParam({ name: 'id', description: '문서 UUID' })
+  @ApiResponse({ status: 200, description: '성공' })
+  @ApiResponse({ status: 404, description: '문서 없음' })
+  async getOne(
+    @CurrentAdmin() admin: AdminContext,
+    @Param('id') id: string,
+  ) {
```

ESLint

[error] Replace `⏎····@CurrentAdmin()·admin:·AdminContext,⏎····@Param('id')·id:·string,⏎··` with `@CurrentAdmin()·admin:·AdminContext,·@Param('id')·id:·string`

(prettier/prettier)